Dockerfile.nginx

```diff
@@ -0,0 +1,2 @@
+FROM nginx:1.11.13-alpine
```

Comment:
It doesn't look to me like we use this in the compose configuration, and we don't document it. I'm a bit confused by it's inclusion.

Is the thinking that this can be used for production? If so, can we document that? Additionally, to keep with some of our conventions, it would be nice to see it named `Dockerfile.production`.